addons/project/migrations/9.0.1.1/pre-migration.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenUpgrade module for Odoo
+#    @copyright 2014-Today: Odoo Community Association
+#    @author: Sylvain LE GAL <https://twitter.com/legalsylvain>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp.openupgrade import openupgrade
+from openerp.addons.openupgrade_records.lib import apriori
+
+
+column_copies = {
+    'project.task': [
+        ('description', None, None),
+    ],
+}
+
+
+@openupgrade.migrate()
+def migrate(cr, version):
+    openupgrade.copy_columns(cr, column_copies)
```

Comment:
Because this field has changed it's type from text to html (see openupgrade_analysis_work.txt).

And because looking at other scripts, it seems to be the way to do it. Now like I said before, I wasn't able to test this.

If there is a better way of doing this, I'd sure like to know as I'm pretty new to all this...